### PR TITLE
Enhancement of convert to numeric on right click

### PR DESCRIPTION
**Pull request by shadrackkibet**
Partially implements #5465.

@africanmathsinitiative/developers this is ready for review.

### CODE REVIEW

**Review by rdstern — CHANGES_REQUESTED** [review]
@shadrackkibet great it works already.  Two small suggestions:
a) Could your non-numeric option be the default.  Currently it is now the second option.
b) Once in the non-numeric dialogue could the variable be selected automatically?
Then it's already ready!

**Comment by shadrackkibet** [comment]
@rdstern the above changes have been done!

**Review by rdstern — APPROVED** [review]
@shadrackkibet great - approving.  I suggest @dannyparsons could do the second review as this was his work originally

File: instat/frmConvertToNumeric.resx [hunk 1]
```diff
@@ -118,10 +118,10 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="lblPossibleSolutions.Text" xml:space="preserve">
-    <value>Some values are non-numeric and will be converted to NA when converting to numeric.
+    <value>Some Values are non-numeric:
```
Comment (dannyparsons) [inline instat/frmConvertToNumeric.resx:121]:
```suggestion
    <value>Some values are non-numeric:
```

File: instat/frmConvertToNumeric.resx [hunk 2]
```diff
@@ -118,10 +118,10 @@
...
 
-- To convert anyway choose "Normal Convert". Non-numeric values will be lost.
-
-- To retain non-numeric values choose "Labelled Convert". All values will be labelled with integers and can be recovered by converting back to character/factor.</value>
+- Choose "Non-numeric" to investigate these values.
```
Comment (dannyparsons) [inline instat/frmConvertToNumeric.resx:123]: Can this button be `Inspect Values`? In general we should use verbs for actions.

**Review by dannyparsons — COMMENTED** [review]
Just a couple of suggestions on the text

File: instat/frmConvertToNumeric.vb [hunk 3]
```diff
@@ -25,6 +26,10 @@
         lblColumnName.Text = strColumnName
     End Sub
 
+    Public Sub SetDataFrameName(strDataFrame As String)
```
Comment (lloyddewit) [inline instat/frmConvertToNumeric.vb:29]: @shadrackkibet I initially thought that `strDataFrameName` should be a public property but then I realized that you'd have to write a `Get` or make the property `WriteOnly`. So on balance, I think what you've done here is a good solution.

**Review by lloyddewit — DISMISSED** [review]

**Review by lloyddewit — APPROVED** [review]

**Review by rdstern — APPROVED** [review]
@shadrack great!

**Pull request merged**